managers/sfrRecord.py

```diff
@@ -442,15 +459,26 @@ def saveItem(self, item):
         return newItem
 
     @staticmethod
-    #Exclude dates in the future and dates before the oldest book publisher company was founded
+    # Exclude dates in the future and dates before the oldest book publisher company was founded
     def publicationDateCheck(edition):
```

Comment:
@mitri-slory In testing this in QA I realized there are actually a few ways in which the `publication_date` can be set when it reaches this method. I've update the function to account for those, which are essentially differently formatted strings.

The core logic that you added is still the core of this, I simply extended the class by handling more cases up front. Would welcome any thoughts that you have on this!